UI/frontend-plugins/decklink-output-ui/DecklinkOutputUI.cpp

```diff
@@ -0,0 +1,69 @@
+#include "DecklinkOutputUI.h"
+#include <obs-module.h>
+#include <util/platform.h>
+#include "decklink-ui-main.h"
+
+DecklinkOutputUI::DecklinkOutputUI(QWidget *parent)
+		: QDialog(parent),
+		  ui(new Ui_Output) {
+	ui->setupUi(this);
+
+	propertiesView = nullptr;
+
+	connect( ui->startOutput, SIGNAL( released() ), this, SLOT(StartOutput()) );
+	connect( ui->stopOutput, SIGNAL( released() ), this, SLOT(StopOutput()) );
+}
+
+void DecklinkOutputUI::ShowHideDialog() {
+	SetupPropertiesView();
+
+	if (!isVisible()) {
```

Comment:
Style:  This `if` statement does not need curly braces.  Remove them.

Is there a particular reason to use `!isVisible()` instead of the _slightly_ shorter `isVisible()`?